packages/controllers/organization_detail.coffee

```diff
@@ -5,7 +5,36 @@ if Meteor.isClient
   Template.organizationDetail.helpers
     organization: ->
       Organizations.findOne(@organizationId)
+    userIsMember: ->
+      UserProfiles.findOne({
+        userId: Meteor.userId(),
+        memberOfOrgs: @organizationId
+      })
+    memberCount: ->
+      Organizations.findOne(@organizationId)?.getMemberProfiles().fetch().length
+    members: ->
+      Organizations.findOne(@organizationId)?.getMemberProfiles()
+
+  Template.organizationDetail.events
+    'click .join-organization' : (event, instance) ->
+      orgId = instance.data.organizationId
+      Meteor.call 'joinOrganization', orgId, (error, response) ->
+        if error
+          toastr.error("Error")
+        else
+          toastr.success("Success")
 
 if Meteor.isServer
-  Meteor.publish 'organizationDetail', (id) ->
-    Organizations.find(id)
+
+  Meteor.publish('organizationDetail', (id) ->
+    [
+      Organizations.find(id)
+      UserProfiles.find({
+        memberOfOrgs: id
+      })
```

Comment:
This will need to hide user emails the same way the profile detail publication does - I wonder if there's a good way to share that code.